girder/constants.py

```diff
@@ -55,10 +55,10 @@
     pass
 
 #: The local directory containing the static content.
-#: Should contain ``clients/web/static``.
-STATIC_ROOT_DIR = ROOT_DIR
-if not os.path.exists(os.path.join(STATIC_ROOT_DIR, 'clients')):
-    STATIC_ROOT_DIR = PACKAGE_DIR
+STATIC_ROOT_DIR = os.environ.get(
+    'GIRDER_STATIC_ROOT',
+    os.path.join(ROOT_DIR, 'clients', 'web', 'static')
+)
```

Comment:
This was done to validate that I can mount an external path for the static assets.  Eventually this will be built into configuration/cli options for both girder's server and `girder-install web`.